Steer/DigitizerWorkflow/src/FDDDigitWriterSpec.h

```diff
@@ -27,17 +27,26 @@ namespace fdd
 template <typename T>
 using BranchDefinition = framework::MakeRootTreeWriterSpec::BranchDefinition<T>;
 
-o2::framework::DataProcessorSpec getFDDDigitWriterSpec()
+o2::framework::DataProcessorSpec getFDDDigitWriterSpec(bool mctruth = true)
 {
   using InputSpec = framework::InputSpec;
   using MakeRootTreeWriterSpec = framework::MakeRootTreeWriterSpec;
-  return MakeRootTreeWriterSpec("FDDDigitWriter",
-                                "fdddigits.root",
-                                "o2sim",
-                                1,
-                                BranchDefinition<std::vector<o2::fdd::Digit>>{InputSpec{"digitBCinput", "FDD", "DIGITSBC"}, "FDDDigit"},
-                                BranchDefinition<std::vector<o2::fdd::ChannelData>>{InputSpec{"digitChinput", "FDD", "DIGITSCH"}, "FDDDigitCh"},
-                                BranchDefinition<o2::dataformats::MCTruthContainer<o2::fdd::MCLabel>>{InputSpec{"labelinput", "FDD", "DIGITLBL"}, "FDDDigitLabels"})();
+  if (mctruth) {
+    return MakeRootTreeWriterSpec("FDDDigitWriter",
+                                  "fdddigits.root",
+                                  "o2sim",
+                                  1,
+                                  BranchDefinition<std::vector<o2::fdd::Digit>>{InputSpec{"digitBCinput", "FDD", "DIGITSBC"}, "FDDDigit"},
+                                  BranchDefinition<std::vector<o2::fdd::ChannelData>>{InputSpec{"digitChinput", "FDD", "DIGITSCH"}, "FDDDigitCh"},
+                                  BranchDefinition<o2::dataformats::MCTruthContainer<o2::fdd::MCLabel>>{InputSpec{"labelinput", "FDD", "DIGITLBL"}, "FDDDigitLabels"})();
```

Comment:
There is a new feature to disable a defined branch, this make the code simpler as the compile time signature can be the same and branches are disabled at runtime by setting number of branches to '0'
```suggestion
                                  BranchDefinition<o2::dataformats::MCTruthContainer<o2::fdd::MCLabel>>{InputSpec{"labelinput", "FDD", "DIGITLBL"}, "FDDDigitLabels", (mctruth ? 1 : 0)})();
```

Reply:
think it should work like that, let me know.

Reply:
Nice. I'll try.